docs/area-owners.md

```diff
@@ -78,6 +78,8 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
 | area-System.DirectoryServices                  | @tquerec      | @tquerec @josephisenhour @joperezr                  |                                                                                                                                                                                                                                                                                           |
 | area-System.Drawing                            | @jeffhandley  | @safern @tannergooding                              |                                                                                                                                                                                                                                                                                           |
 | area-System.Dynamic.Runtime                    | @jaredpar     | @cston @333fred                                     | Archived component - limited churn/contributions (see [#27790](https://github.com/dotnet/runtime/issues/27790))                                                                                                                                                                            |
+| area-System.Formats.Asn1                       | @jeffhandley  | @bartonjs                                           |
+| area-System.Formats.Cbor                       | @jeffhandley  | @eiriktsarpalis                                     |
```

Comment:
@bartonjs -- can you be a secondary owner of Cbor?
@eiriktsarpalis and @GrabYourPitchforks -- can you be additional owners of Asn1?

Reply:
OK by me

Reply:
Sure, I can be an ASN.1 alt.

Reply:
I wouldn't mind being the primary owner either.